src/utils/vacation-days-utils.tsx

```diff
@@ -0,0 +1,43 @@
+import { Typography, Box} from "@mui/material";
+import { theme } from "../theme";
+import { Person } from "../generated/client";
+import strings from "../localization/strings";
+import { useState } from "react";
+import { personsAtom } from "../atoms/person";
+import { useAtom } from "jotai";
+
+/**
+ * Display persons vacation days
+ * @param Person timebank person
+ */
+export const renderVacationDays = (person: Person) => {
+const loading = useState(true);
```

Comment:
I don't think this is correct. If the state is not being changed then it could just be a regular variable rather than state, also as it is not being changed this means the condition below if (!loading && persons.length) doesn't really make sense. Do you still need loading here at all or can it be handled in the parent component e.g. vacations screen and card, where this render function is being called

Reply:
Made loading into a regular variable. If it's complitly removed, it fails to fetch the persons data.